### PR TITLE
Dockerfile lint 2

### PR DESCRIPTION
**Pull request by ammarlakis**
**Description**

Changes proposed in this pull request:

- Fixing linting errors in Dockerfiles. Patch 2 includes the components:
  - components/busola-migrator/Dockerfile
  - components/central-application-connectivity-validator/Dockerfile
  - components/central-application-gateway/Dockerfile
  - components/compass-runtime-agent/Dockerfile
  - components/compass-runtime-agent/hack/Dockerfile
  - components/connection-token-handler/Dockerfile
  - components/connector-service/Dockerfile
  - components/function-controller/deploy/jobinit/Dockerfile
  - components/function-controller/deploy/manager/Dockerfile
  - components/function-controller/deploy/webhook/Dockerfile
  - components/function-runtimes/nodejs12/Dockerfile
  - components/function-runtimes/nodejs14/Dockerfile
  - components/function-runtimes/python38/Dockerfile
  - components/function-runtimes/python39/Dockerfile
  - tests/connection-token-handler-tests/Dockerfile
  - tests/connector-service-tests/Dockerfile
  - tests/end-to-end/upgrade/Dockerfile
  - tests/function-controller/Dockerfile

**Related issue(s)**
Related to #12854

### CODE REVIEW

**Comment by ammarlakis** [comment]
/test pre-main-kyma-protecode-guard
--


<br class="Apple-interchange-newline">

File: components/function-runtimes/python38/Dockerfile [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 FROM alpine:3.14.2
 
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
```
Comment (dbadura) [inline components/function-runtimes/python38/Dockerfile:12]: ash?

Comment (ammarlakis) [reply components/function-runtimes/python38/Dockerfile:12]: It's the shell for alpine. I'm following the recommendation from hadolint tool https://github.com/hadolint/hadolint/wiki/DL4006#set-the-shell-option--o-pipefail-before-run-with-a-pipe-in.

**Review by Maladie — DISMISSED** [review]

File: resources/serverless/values.yaml [hunk 2]
```diff
@@ -77,7 +77,7 @@ global:
       directory: "tpi"
     function_controller:
       name: "function-controller"
-      version: "PR-13074"
+      version: "PR-12963"
     function_webhook:
       name: "function-webhook"
       version: "PR-13074"
```
Comment (dbadura) [inline resources/serverless/values.yaml:83]: please bump also this

File: resources/serverless/values.yaml [hunk 3]
```diff
@@ -86,16 +86,16 @@ global:
...
```
Comment (dbadura) [inline resources/serverless/values.yaml:86]: and this ;)

**Review by dbadura — APPROVED** [review]

**Review by franpog859 — CHANGES_REQUESTED** [review]
Please, bump also Application Operator

**Review by franpog859 — APPROVED** [review]

**Review by dbadura — APPROVED** [review]

**Pull request merged**